nilearn/input_data/tests/test_surf_labels_masker.py

```diff
@@ -0,0 +1,242 @@
+import os
+import tempfile
+import numpy as np
+import nibabel as nb
+
+from nilearn._utils.exceptions import DimensionError
+
+import nibabel
+import pytest
+
+from nilearn.input_data.surf_labels_masker import (SurfLabelsMasker,
+                                                   _all_files_check,
+                                                   _get_single_surf,
+                                                   surf_to_signals_labels)
+from numpy.testing import assert_array_equal
+
+
+def test_surf_labels_maskers():
+
```

Comment:
Instead of writing multiplle tests within a single function, and separating them with comments, it is better to write multiple small tests, with the test function names reflecting what is being tested (instead of the comment).

Reply:
If that means recreating the simple data for testing, that is okay.